notebooks/py/assay_comparison.py

```diff
@@ -0,0 +1,237 @@
+### Load Decoded Images
+# EPY: END markdown
+
+# EPY: START markdown
+#Numpy load can't download files from s3 either.
+# EPY: END markdown
+
+# EPY: START code
+merfish_link = data_root + "merfish_decoded_image.npy"
+dartfish_link = data_root + "dartfish_decoded_image.npy"
+
+merfish_npy = tmp + "merfish_decoded_image.npy"
+dartfish_npy = tmp + "dartfish_decoded_image.npy"
+
+# EPY: ESCAPE !curl -o $merfish_npy $merfish_link
+# EPY: ESCAPE !curl -o $dartfish_npy $dartfish_link
+
+
+merfish_decoded_image = np.squeeze(np.load(merfish_npy))
+dartfish_decoded_image = np.squeeze(np.load(dartfish_npy))
+# EPY: END code
+
+# EPY: START markdown
+### Show Different Background Types for MERFISH
+# EPY: END markdown
+
+# EPY: START code
+f, ((ax1, ax2), (ax3, ax4), (ax5, ax6)) = plt.subplots(ncols=2, nrows=3, figsize=(30, 45))
+spots(merfish, background_image=np.zeros_like(merfish_dots), spots_kwargs=dict(alpha=1.), ax=ax1)
+spots(merfish, background_image=merfish_dots, spots_kwargs=dict(alpha=1.), ax=ax3)
+spots(merfish, background_image=merfish_nuclei, spots_kwargs=dict(alpha=1.), ax=ax5)
+spots(decoded_image=merfish_decoded_image, decoded_image_kwargs=dict(alpha=1.), ax=ax2)
+spots(decoded_image=merfish_decoded_image, background_image=merfish_dots, decoded_image_kwargs=dict(alpha=1.), ax=ax4)
+spots(decoded_image=merfish_decoded_image, background_image=merfish_nuclei, decoded_image_kwargs=dict(alpha=1.), ax=ax6);
+# EPY: END code
+
+# EPY: START markdown
+#From these examples, we can see that the point cloud over-estimates the spot size (perhaps we're calculating radius wrong?)
+# EPY: END markdown
+
+# EPY: START markdown
+### Show different Dartfish Spots Plots
+# EPY: END markdown
+
+# EPY: START code
+f, ((ax1, ax2), (ax3, ax4), (ax5, ax6)) = plt.subplots(ncols=2, nrows=3, figsize=(30, 45))
+spots(dartfish, background_image=np.zeros_like(dartfish_dots), spots_kwargs=dict(alpha=1.), ax=ax1)
+spots(dartfish, background_image=dartfish_dots, spots_kwargs=dict(alpha=1.), ax=ax3)
+spots(dartfish, background_image=dartfish_nuclei, spots_kwargs=dict(alpha=1.), ax=ax5)
+spots(decoded_image=dartfish_decoded_image, decoded_image_kwargs=dict(alpha=1.), ax=ax2)
+spots(decoded_image=dartfish_decoded_image, background_image=dartfish_dots, decoded_image_kwargs=dict(alpha=1.), ax=ax4)
+spots(decoded_image=dartfish_decoded_image, background_image=dartfish_nuclei, decoded_image_kwargs=dict(alpha=1.), ax=ax6);
+# EPY: END code
+
+# EPY: START markdown
+### Show different background types for ISS
+# EPY: END markdown
+
+# EPY: START code
+f, ((ax1, ax2), (ax3, ax4), (ax5, ax6)) = plt.subplots(ncols=2, nrows=3, figsize=(30, 40))
+spots(iss, background_image=np.zeros_like(iss_dots), spots_kwargs=dict(alpha=1.), ax=ax1)
+spots(iss, background_image=iss_dots, spots_kwargs=dict(alpha=1.), ax=ax3)
+spots(iss, background_image=iss_nuclei, spots_kwargs=dict(alpha=1.), ax=ax5)
+
+# ISS doesn't have a decoded image, so leave those plots blank to make this clear
```

Comment:
ISS can have a decoded image though -- if we run a version of the pipeline that uses pixel detection instead of spot detection. we should do that, it will be interesting to comare the threeway of 1. benchmark 2. iss spot detection 3. iss pixel detection. it will also complete the missing column of this plot, thus showing off the flexibility of the code. although this could be it's own separate plot as well w/ 2 or 3 copy number comparison plots: bench vs. spots, bench vs. pixels, spots vs. pixels